tests/unit/store/playback.spec.js

```diff
@@ -124,71 +123,81 @@ describe("store/playback", () => {
 
         store.commit("playback/stop_playback_progression");
 
-        expect(spied_clear_interval).toBeCalledWith(expected_interval_id);
+        expect(spied_clear_interval).toHaveBeenCalledWith(expected_interval_id);
         expect(
           store.state.playback.playback_progression_interval_id
         ).toBeNull();
       });
     });
 
-    test("When set_playback_state is committed the playback_state is mutated", () => {
+    test("When set_playback_state is committed, Then the playback_state is mutated", () => {
       store.commit(
         "playback/set_playback_state",
         playback_module.ENUMS.PLAYBACK_STATES.PLAYING
       );
-      expect(store.state.playback.playback_state).toEqual(
+      expect(store.state.playback.playback_state).toStrictEqual(
         playback_module.ENUMS.PLAYBACK_STATES.PLAYING
       );
       store.commit(
         "playback/set_playback_state",
         playback_module.ENUMS.PLAYBACK_STATES.STOPPED
       );
-      expect(store.state.playback.playback_state).toEqual(
+      expect(store.state.playback.playback_state).toStrictEqual(
         playback_module.ENUMS.PLAYBACK_STATES.STOPPED
       );
     });
 
-    it("should set is_playing to false initially", () => {
-      expect(store.getters["playback/is_playing"]).toBe(false);
+    test("When the playback store is initialized, Then is_playing is set to false", () => {
+      expect(
+        store.state.playback.playback_state ===
```

Comment:
if you removed the getter `is_playing` from the code, then you can't be writing tests with titles talking about `is_playing`. You should talk about the playback state being PLAYING or STOPPED or some other value depending on the test